input/pagecontent/2-5-EventNotification.md

```diff
@@ -100,9 +81,9 @@ Field | Optionality | Type | Description
 }
 ```
 
-### `webhook` and `websocket` Event Notification Sequence
+### Event Notification Sequence
 
-{% include img.html img="EventNotificationSequence.png" caption="Figure: Event Notification flow diagram" %}
+{%include EventNotificationSequence.svg%}

```

Comment:
<?xml version=”1.0” encoding=”UTF-8” standalone=”no”?> seems to be showing up at the top of each svg?

Reply:
Agree, revisit after merge.